### PR TITLE
[Query] Fix resource leak

### PR DESCRIPTION
**Pull request by gichan-jang**
Fix resource leaks reported by coverity.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ ]Passed [ ]Failed [ *]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_query/tensor_query_common.c [hunk 1]
```diff
@@ -206,6 +206,7 @@ nnstreamer_query_connect (TensorQueryProtocol protocol, const char *ip,
     }
     default:
       nns_loge ("Unsupported protocol.");
+      nnstreamer_query_close (conn);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_query/tensor_query_common.c:209]: you also have to free `conn->host` on failure

File: gst/nnstreamer/tensor_query/tensor_query_common.c [hunk 2]
```diff
@@ -567,13 +568,15 @@ accept_socket_async_cb (GObject * source, GAsyncResult * result,
   saddr = g_socket_get_remote_address (socket, &err);
   if (!saddr) {
     nns_loge ("Failed to get socket address: %s", err->message);
+    g_free (conn);
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_query/tensor_query_common.c:571]: need to free `err`? line 558 as well.

Comment (again4you) [reply gst/nnstreamer/tensor_query/tensor_query_common.c:571]: Right!. `g_clear_error()` should be needed after using `GError`.

**Review by again4you — CHANGES_REQUESTED** [review]

File: gst/nnstreamer/tensor_query/tensor_query_common.c (hunk 2 above)
Comment (gichan-jang) [reply gst/nnstreamer/tensor_query/tensor_query_common.c:575]: Thanks. Updated!

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by again4you — APPROVED** [review]
LGTM!

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**